src/editingcontroller.js

```diff
@@ -6,12 +6,157 @@
+		 */
 		this.view = new ViewDocument();
+
+		// Attach default observers.
+		this.view.addObserver( MutationObserver );
+		this.view.addObserver( SelectionObserver );
+		this.view.addObserver( FocusObserver );
+		this.view.addObserver( KeyObserver );
+
+		/**
+		 * Mapper which describe model-view binding.
+		 *
+		 * @readonly
+		 * @member {engine.conversion.Mapper} engine.EditingController#mapper
+		 */
+		this.mapper = new Mapper();
+
+		// Convert view selection to model.
+		this._listenter.listenTo( this.view, 'selectionChange', convertSelectionChange( model, this.mapper ) );
+
+		/**
+		 * Model to view conversion dispatcher, which converts changes from the model to
+		 * {@link engine.EditingController#view editing view}.
+		 *
+		 * To attach model to view converter to the editing pipeline you need to add lister to this property:
+		 *
+		 *		editing.modelToView( 'insert:$element', customInsertConverter );
+		 *
+		 * Or use {@link engine.conversion.ModelConverterBuilder}:
+		 *
+		 *		BuildModelConverterFor( editing.modelToView ).fromAttribute( 'bold' ).toElement( 'b' );
+		 *
+		 * @readonly
+		 * @member {engine.conversion.ModelConversionDispatcher} engine.EditingController#modelToView
+		 */
+		this.modelToView = new ModelConversionDispatcher( {
+			writer: this.view.writer,
+			mapper: this.mapper,
+			viewSelection: this.view.selection
+		} );
+
+		this._listenter.listenTo( this.model, 'change', ( evt, type, changes ) => {
+			this.modelToView.convertChange( type, changes );
+		} );
+
+		this._listenter.listenTo( this.model, 'changesDone', () => {
+			this.modelToView.convertSelection( model.selection );
+			this.view.render();
+		} );
+
+		// Attach default content converters.
+		this.modelToView.on( 'insert:$text', insertText() );
+		this.modelToView.on( 'remove', remove() );
+		this.modelToView.on( 'move', move() );
+
+		// Attach default selection converters.
+		this.modelToView.on( 'selection', clearAttributes() );
+		this.modelToView.on( 'selection', convertRangeSelection() );
+		this.modelToView.on( 'selection', convertCollapsedSelection() );
 	}
 
-	destroy() {}
+	/**
+	 * {@link engine.view.Document#createRoot Creates} a view root and {@link engine.conversion.Mapper#bindElements binds}
+	 * the model root with view root and and view root with DOM element:
+	 *
+	 *		editing.createRoot( document.querySelector( div#editor ) );
+	 *
+	 * If the DOM element is not available at the time you want to create a view root, for instance it is iframe body
+	 * element, it is possible to create view element and bind the DOM element later:
+	 *
+	 *		editing.createRoot( 'body' );
+	 *		editing.view.attachDomRoot( iframe.contentDocument.body );
+	 *
+	 * @param {Element|String} domRoot DOM root element or the name of view root element if the DOM element will be
+	 * attached later.
+	 * @param {String} [name='main'] Root name.
+	 * @returns {engine.view.ContainerElement} View root element.
+	 */
+	createRoot( domRoot, name = 'main' ) {
```

Comment:
We agreed that it's obvious that editing controller can only modify the view stuff.